plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/navigation-extra-screen/sidebar-navigation-extra-screen.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { useQuery } from '@woocommerce/navigation';
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { SidebarPatternScreen } from '../pattern-screen/sidebar-pattern-screen';
+
+import './style.scss';
+import { PATTERN_CATEGORIES } from '../pattern-screen/categories';
+
+const getPatternCategoryByPath = ( path: string ) => {
+	// Define the regex pattern to match everything after "/homepage/"
+	const regex = /\/homepage\/(.*)/;
+
+	// Apply the regex pattern to the pathname part of the URL
+	const match = path.match( regex );
+
+	// If a match is found, return the captured group (everything after "/homepage/")
+	if ( match && match[ 1 ] ) {
+		return match[ 1 ];
+	}
+
+	const pattern_category = match && match[ 1 ];
```

Comment:
Likely not intentional, as `pattern_category` will always be false in this context!

Reply:
Nice catch! Fixed with: 8ec4ae631fe9f67998fa1d67bec72b6f1d2b0a67